packages/itwinui-css/src/slider/slider.scss

```diff
@@ -20,7 +19,8 @@ $tick-height: $iui-baseline; // 12px
 
   svg {
     @include iui-icon-style('m');
-    margin-top: $iui-xs;
+    @include iui-icon-color;
```

Comment:
Is this color change ok?
![image](https://user-images.githubusercontent.com/36186912/185616575-69de0bfe-dd58-4c94-8cc4-6188d952ee6f.png)

Reply:
I'm not sure, but I think the grey one is the correct color. That is how it looks on the website. It was also grey when I was working on the [vertical slider PR](https://github.com/iTwin/iTwinUI/pull/707)